modules/solid_mechanics/src/materials/NewtonIterationSingleVariable.C

```diff
@@ -0,0 +1,443 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+#include "NewtonIterationSingleVariable.h"
+
+#include "InputParameters.h"
+#include <cmath>
+#include "Conversion.h"
+
+template <>
+InputParameters
+validParams<NewtonIterationSingleVariable>()
+{
+  InputParameters params = emptyInputParameters();
+
+  // Newton iteration control parameters
+  params.addParam<unsigned int>("max_its", 200, "Maximum number of Newton iterations");
```

Comment:
There were enough cases in the BISON assessment cases that failed with a lower number that I decided to bump it up.  Those are usually cases that have crazy predictor stresses on the order of 1e100, and they actually converge if you give them 200 iterations.  The old method computed the residual relative to the initial residual, so it could happily accept a solution with a residual on the order of 1e100.